### ISSUE TITLE
Broken hamburger menu in the page header

### ISSUE THREAD

**Issue by SebinSong** [okTurtles/groupincome.org#72]
https://github.com/okTurtles/groupincome.org/assets/17641213/58d0a314-d3c1-4723-9558-24a235bee875

Navigation menu in the header becomes a hamburger menu in non-desktop size screen. but Clicking on it doesn't reveal the menu. The navigation menu in narrow screen needs to be implemented or a design for it needs to be created if there already isn't.

**Comment by neshvig10** [comment]
Can I fix this issue ?

**Comment by taoeffect** [comment]
@neshvig10 If you'd like to work on this, sure, however, as Sebin mentioned we don't have designs for this yet, and any work you do that conflicts with the designs will need to be redone.

I've created a comment on Figma about this and tagged our designer @leihla: https://www.figma.com/file/TsreEdGTmMweBlZr9eqgIu?node-id=3696:6537&mode=design#491139182

If you think you have a good idea of how this should behave, feel free to go for it. I might be misremembering, but it's possible @pieer even had already implemented part of this at some point.

We can go with a temporary solution while we're waiting on the designs, so I've assigned this to you. We're always happy to work with volunteers and see how we can make things work out with them. 🙂
